var/spack/repos/builtin/packages/py-scikit-learn/package.py

```diff
@@ -46,8 +47,6 @@ class PyScikitLearn(PythonPackage):
     version("0.22.1", sha256="51ee25330fc244107588545c70e2f3570cfc4017cff09eed69d6e1d82a212b7d")
     version("0.22", sha256="314abf60c073c48a1e95feaae9f3ca47a2139bd77cebb5b877c23a45c9e03012")
 
-    variant("openmp", default=True, description="Build with OpenMP support")
```

Comment:
OpenMP has been required for a while now, not sure when that happened. Let's just drop the variant.